src/middlewares/checkPostAndJob.js

```diff
@@ -0,0 +1,37 @@
+import { Post, Job } from '../models';
+import * as statusCodes from '../constants/statusCodes';
+import { notExist } from '../constants/responseMessages';
+
+const checkPostAndJob = async (req, res, next) => {
```

Comment:
@gracelungu Why this middleware when we already have `checkPost` and `checkJob`

Reply:
I didn't want to modify the other middlewares and I needed to check in the two models before returning the response if none of Jobs or Posts is found or doing next